src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useContext } from 'react';
+
+import { TodosFilter } from './TodosFilter';
+import { ToDoContext } from '../Context/ToDoContext';
+
+export const Footer: React.FC = () => {
+  const { todos, setTodos } = useContext(ToDoContext);
+
+  const unCompleted = [...todos].filter(todo => todo.completed === false);
+  const completed = [...todos].filter(todo => todo.completed === true);
```

Comment:
```suggestion
  const uncompleted = todos.filter(todo => !todo.completed);
  const completed = todos.filter(todo => todo.completed);
```
method filter returns new array